MaxText/layers/normalizations.py

```diff
@@ -17,35 +17,68 @@
 from typing import Any, Tuple, Optional
 
 from flax import linen as nn
+from flax import nnx
 from jax import lax
 import jax.numpy as jnp
 from MaxText.layers import initializers
 
 Initializer = initializers.Initializer
 
 
-class RMSNorm(nn.Module):
+class RMSNorm(nnx.Module):
   """RMS normalization."""
 
-  epsilon: float = 1e-6
-  dtype: Any = jnp.float32
-  weight_dtype: Any = jnp.float32
-  kernel_axes: Tuple[Optional[str], ...] = ()
-  scale_init: Initializer = nn.initializers.ones
+  def __init__(
+      self,
+      features: int,
```

Comment:
Is it possible to make this more descriptive like in https://github.com/AI-Hypercomputer/maxtext/pull/1840?

Reply:
Changing it to `num_features`.

Reply:
Thanks @cgarciae. As a nit, should it be `features_shape` to match the style from #1840?

Reply:
In this case its not a shape. I'm also not super convinced about the shape semantics in `DenseGeneral` because you can also pass and `int`.